src/services/tx/extractTxInfo.ts

```diff
@@ -74,12 +78,21 @@ const extractTxInfo = (
         } else {
           return txDetails.txInfo.transferInfo.tokenAddress
         }
+      case 'SwapOrder':
+        const swapOrderTo = txDetails.txData?.to.value
+        // TODO: Remove assertion after type is corrected
```

Comment:
Is it corrected now?

Reply:
I don't know. According to the types txData might not exist. If it exist, then it definetely has a to prop. I would leave the typeguard check for now. (I've removed locally and the app has not crashed, but would prefer to have it there a bit longer)